projects/kit/components/input-slider/input-slider.template.html

```diff
@@ -35,7 +36,7 @@
     [max]="computedSteps"
     [segments]="segments"
     [keySteps]="computeKeySteps(keySteps, min, max)"
-    [attr.disabled]="readOnly || disabled || null"
+    [attr.disabled]="readOnly || computedDisabled || null"
```

Comment:
Why not `[disabled]` input? Should work with `tuiSlider` + `ngModel`.

Reply:
If I have not mistaken, just `[disabled]` doesn't add `:disabled` pseudo-class for slider